towerlib/entities/job.py

```diff
@@ -793,8 +823,22 @@ def project(self):
             Inventory: The project that the job template is part of.
 
         """
+        if not self._data.get('related', {}).get('project'):
```

Comment:
This is slightly unelegant with traversing the same data twice. Please code it like :
```
url = self._data.get('related', {}).get('project')
if not url:
    return None
return self._tower._get_object_by_url('Project', url) # pylint: disable=protected-access
```